resources/views/applicant/strategic_response_application/step_01/questions.html.twig

```diff
@@ -4,135 +4,135 @@
     Applicant: Application Post - Questions
 
 ============================================================================= #}
-<div
```

Comment:
I think I made a few changes to this file since you branched off feature/strategic-response-application. I'll try to reconcile them.

Reply:
Hokay.